From 5974d4ba65fc5ad9253857bf992ffd3c635baf2a Mon Sep 17 00:00:00 2001
From: Colin Watson <cjwatson@ubuntu.com>
Date: Mon, 23 Mar 2015 14:27:41 +0000
Subject: [PATCH 370/506] hostfs: Drop unnecessary feature test macros

_BSD_SOURCE was added to allow the use of DT_DIR, but that was removed
in e768b77068a0b030a07576852bd0f121c9a077eb.  While adding
_DEFAULT_SOURCE as well works around problems with current glibc,
neither is in fact needed nowadays.
---
 grub-core/kern/emu/hostfs.c | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/grub-core/kern/emu/hostfs.c b/grub-core/kern/emu/hostfs.c
index 823116d..7b28c00 100644
--- a/grub-core/kern/emu/hostfs.c
+++ b/grub-core/kern/emu/hostfs.c
@@ -19,11 +19,6 @@
 
 #include <config-util.h>
 
-/* Legacy feature macro.*/
-#define _BSD_SOURCE
-/* New feature macro that provides everything _BSD_SOURCE and
- *    _SVID_SOURCE provided and possibly more.  */
-#define _DEFAULT_SOURCE
 #include <grub/fs.h>
 #include <grub/file.h>
 #include <grub/disk.h>
-- 
2.4.3

